docker/compose/docker-compose-nfs.yml

```diff
@@ -0,0 +1,91 @@
+#
```

Comment:
Why do we have `docker-compose-nfs.yml` and `docker-compose-nfs-mount.yml`? They seem to be the same file. I also don't understand why we are fixing the number of bookies here to three and how is that related to mounting an NFS volume?